content/en/docs/v3.5/install.md

```diff
@@ -71,8 +71,49 @@ source by following these steps:
 
 ## Installation via OS packages
 
+There are various ways of installing etcd on different operating systems and these are just some examples how it can be done.
+
+### MacOS (Homebrew)
+
+1. Update homebrew:
+```sh
+$ brew update
+```
+
+2. Install etcd:
+```sh
+$ brew install etcd
+```
+
+3. Verify install 
+```sh
+$ etcd --version
+```
+
+### Debian Linux (apt)
```

Comment:
I would not want to recommend to users unsupported and unmaintained way to install etcd. Let's drop it or improve the situation.

Reply:
Just for everyone's info there has been some discussion earlier this year on getting an updated etcd package in but it looks like it has stalled: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=991462 

We can remove it for now sure. I'll keep an eye on it and if the situation improves I can raise a pr.